packages/app/src/messages/components/messages-panel/messages-panel.interface.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { Thread, Message } from "App/messages/dto"
+import { SearchResult } from "App/search/dto"
+
+export interface MessagesPanelProps {
+  searchValue: string
+  onSearchValueChange: (value: string) => void
+  onNewMessageClick: () => void
+  buttonDisabled: boolean
+  selectedIds: string[]
+  allItemsSelected: boolean
+  toggleAll: (event: React.ChangeEvent<Element> | undefined) => void
+  onDeleteClick: () => void
+  onSearchEnterClick: () => void
+  onSelect: (record: Thread & Message) => void
```

Comment:
Would be good to apply the same as in https://github.com/mudita/mudita-center/pull/945#discussion_r958503690

Reply:
https://github.com/mudita/mudita-center/pull/945/commits/735a58b6787a26a9f9fc3714fa5138d4dc264f41